apps/lfx-one/src/app/modules/profile/individual-enrollment/profile-individual-enrollment.component.html

```
@@ -31,92 +31,128 @@
               }
             </div>
-          } @else if (item.price !== null && item.price !== undefined) {
-            <div class="flex flex-wrap items-center gap-4 text-sm text-gray-600" data-testid="individual-enrollment-details">
-              <span
-                ><span class="font-medium">${{ item.price }}</span> / year</span
-              >
-            </div>
           }
 
           <!-- Benefits list -->
           @if (item.benefits && item.benefits.length > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)